atomic_reactor/inner.py

```diff
@@ -312,6 +314,9 @@ def base_image_inspect(self):
             self._base_image_inspect = self.builder.tasker.inspect_image(self.builder.base_image)
         return self._base_image_inspect
 
+    def throw_cancelled_build_exception(self, *args, **kwargs):
```

Comment:
**NITPICK ALERT** I know it's silly, but I can never remember how to spell canceled (1 L vs 2 LLs), can we at least make `throw_cancelled_build_exception` and `BuildCanceledException` consistent ? :)